lib/features/home/view/widgets/property_item_view.dart

```diff
@@ -0,0 +1,376 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:hatspace/dimens/hs_dimens.dart';
+import 'package:hatspace/features/home/data/property_item_data.dart';
+import 'package:hatspace/gen/assets.gen.dart';
+import 'package:hatspace/strings/l10n.dart';
+import 'package:hatspace/theme/hs_theme.dart';
+
+class PropertyItemView extends StatelessWidget {
+  final PropertyItemData property;
+  const PropertyItemView({required this.property, super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      children: [
+        _PropertyImgsCarousel(
+          photos: property.photos,
+          ownerName: property.ownerName,
+          ownerAvatar: property.ownerAvatar,
+          availableDate: property.availableDate,
+          isFavorited: property.isFavorited,
+        ),
+        Container(
+          padding: const EdgeInsets.symmetric(vertical: HsDimens.spacing8),
+          child: Column(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: [
+              Row(
+                children: [
+                  Expanded(
+                      child: Row(
+                    children: [
+                      Flexible(
+                          child: Text(
+                        property.price,
+                        maxLines: 4,
```

Comment:
@hophongit98 Because price is important information, so specifying maxlines = 4 to cover the case when user uses smaller device. On the other hand, i don't want it to be as big as it want so the UI may gone bad. Btw, what is your suggestion ?

Reply:
1 line